pages/common/redis-server.md

```diff
@@ -0,0 +1,16 @@
+# redis-server
+
+> Persistent key-value database.
+> More information: <https://redis.io/>.
+
+- Start redis daemon, in the foreground, with the default port (6379):
```

Comment:
```suggestion
- Start Redis server in the default port (6379) and write logs to `stdout`:
```

Reply:
I made a change consistent with your suggestion, but I slightly changed the english around the port number.